app/server/lib/gristSessions.ts

```diff
@@ -3,14 +3,15 @@ import {parseSubdomain} from 'app/common/gristUrls';
 import {isNumber} from 'app/common/gutil';
 import {RequestWithOrg} from 'app/server/lib/extractOrg';
 import {GristServer} from 'app/server/lib/GristServer';
+import log from 'app/server/lib/log';
 import {fromCallback} from 'app/server/lib/serverUtils';
 import {Sessions} from 'app/server/lib/Sessions';
 import {promisifyAll} from 'bluebird';
 import * as crypto from 'crypto';
 import * as express from 'express';
 import assignIn = require('lodash/assignIn');
 import * as path from 'path';
-
+import { createClient } from 'redis';
```

Comment:
Nit:
```suggestion
import {createClient} from 'redis';
```

I remember somewhere we had to be consistent in a single file about the use of spaces inside brackets, to allow grsit-desktop to build. That might not be the case anymore.

Reply:
Thanks :)